### PR TITLE
Np 1979 join resource with doi request

### PR DESCRIPTION
**Pull request by axthosarouris**

### CODE REVIEW

File: storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoTest.java [hunk 1]
```diff
@@ -11,72 +10,55 @@
...
-    Javers javers = JaversBuilder.javers().build();
+public class DaoTest extends ResourcesDynamoDbLocalTest {
 
     @BeforeEach
     public void init() {
         super.init();
     }
 
-    @ParameterizedTest(name = "get Type Returns Name Of The Contained Object: {0}")
+    @ParameterizedTest(name = "getTypeReturnsNameOfTheContainedObject: {0}")
```
Comment (brinxmat) [inline storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoTest.java:45]: Were these changes intentional?

File: storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoUtils.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    public static final String EMPTY_VALUE_ERROR = "ValueMap was either null or empty";
+    public static ResourceTest resourceGenerator = new ResourceTest();
+
+    public static Resource sampleResource() throws InvalidIssnException, MalformedURLException {
+        return Resource.fromPublication(resourceGenerator.samplePublication(
+            resourceGenerator.sampleJournalArticleReference()));
+    }
+
+    public static ResourceDao resourceDao() throws InvalidIssnException, MalformedURLException {
```
Comment (brinxmat) [inline storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoUtils.java:33]: The naming here is somewhat unclear…generateResourceDao?

File: storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoUtils.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+import no.unit.nva.publication.storage.model.RowLevelSecurity;
+import no.unit.nva.publication.storage.model.WithIdentifier;
+import nva.commons.core.attempt.Try;
+
+public final class DaoUtils {
+
+    public static final String EMPTY_VALUE_ERROR = "ValueMap was either null or empty";
+    public static ResourceTest resourceGenerator = new ResourceTest();
+
+    public static Resource sampleResource() throws InvalidIssnException, MalformedURLException {
```
Comment (brinxmat) [inline storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoUtils.java:28]: generateSampleResource?

File: storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoUtils.java (hunk 2 above)
Comment (brinxmat) [reply storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoUtils.java:33]: or following previous sampleResourceDao?

**Review by brinxmat — APPROVED** [review]

File: storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoTest.java (hunk 1 above)
Comment (axthosarouris) [reply storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/DaoTest.java:45]: Yes. I thought to have use the same name as the test function. Maybe not the best idea.

**Review by brinxmat — APPROVED** [review]

**Review by janbj — APPROVED** [review]
